README.md

```diff
@@ -44,7 +44,7 @@ pip install -r requirements.txt
 
 1. Activate Python environment. From the root directory of this repository, run
 ```
-source mt-fast-training/bin/activate
+source fast-training/bin/activate
```

Comment:
I installed the new repo, and I get the following error when I run these lines:

>>source .env
readlink: illegal option -- f
usage: readlink [-n] [file ...]

I get this error on my mac but not my Linux machine

Reply:
I will find someone with a mac to test it. I made a change that hopefully works.